addons/sms/static/tests/sms_widget_test.js

```diff
@@ -8,18 +8,24 @@ var testUtils = require('web.test_utils');
 
 var createView = testUtils.createView;
 
+const PHONE = "+32494444444";
```

Comment:
there is no need for this.
We prefer to have basically no data as global variables. If you want to declare constants for one test you should do it in that test.

Before each test, all data is reset with the beforeEach function. It guarantees that each tests starts from scratch data that you can fully control from within the test. All data is put on the `this` of the test function.

```
Qunit.test('sometitle', () => {
     this.data

})
```